src/stylesheets/components/_accordions.scss

```diff
@@ -90,3 +90,7 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-bottom: 0;
   }
 }
+
+.usa-accordion-content-collapsed {
```

Comment:
@maya - Saw that. It's  good solution. Incorporated it here. Also tested just doing `height: auto`, which did work - no JS changes needed. I do like the height solution from the screen-reader perspective - though I'm not sure if `visibility: hidden` would cause it to be skipped. Which brings up the question: Do we want accordion content to be hidden from screen-readers?